### PR TITLE
fix bnpcs not despawning

### PR DESCRIPTION
**Pull request by NotAdam**

### CODE REVIEW

File: src/world/Actor/BNpc.cpp [hunk 1]
```diff
@@ -306,6 +311,10 @@ void Sapphire::Entity::BNpc::update( int64_t currTime )
 
     case BNpcState::Idle:
     {
+      // passive mobs should ignore players unless aggro'd
+      if( m_aggressionMode == 1 )
```
Comment (takhlaq) [inline src/world/Actor/BNpc.cpp]: we should enum these at some point

File: src/world/Network/PacketWrappers/NpcSpawnPacket.h [hunk 2]
```diff
@@ -76,17 +76,14 @@ namespace Sapphire::Network::Packets::Server
       m_data.bNPCName = bnpc.getBNpcNameId();
 
       m_data.state = 1;
```
Comment (takhlaq) [inline src/world/Network/PacketWrappers/NpcSpawnPacket.h]: this too

File: src/world/Actor/BNpc.cpp (hunk 1 above)
Comment (takhlaq) [reply src/world/Actor/BNpc.cpp]: in fact, i think there's already enums for these in exds

Comment (NotAdam) [reply src/world/Actor/BNpc.cpp]: I can't find it in the exds, I know it was there at one point but looked before making the PR and couldn't find it

maybe i'm blind?

**Pull request merged**